services/QuillLMS/client/app/bundles/Teacher/components/subscriptions/current_subscription.jsx

```diff
@@ -197,15 +234,21 @@ export default class CurrentSubscription extends React.Component {
   }
 
   nextPlanAlert(body) {
-    return <div className="next-plan-alert flex-row vertically-centered"><i className="fas fa-icon fa-lightbulb-o" />{body}</div>;
+    return (
+      <div className="next-plan-alert flex-row vertically-centered">
+        <i className="fas fa-icon fa-lightbulb-o" />
+        {body}
+      </div>
+    )
   }
 
   nextPlanAlertOrButtons(condition, renewDate) {
-    const { lastFour, } = this.state
     const { authorityLevel, subscriptionStatus, } = this.props
+    const { last_four } = subscriptionStatus
```

Comment:
I'm noticing more mixed variable naming (some `camelcase`, some `snakecase`). Maybe this is coming from Stripe, so we're not in control of it, but it might be worth discussing with @emilia-friedberg and @eadams17 how we should determine which to use, so that if someone else is working on this code, it's clear what to use (or it can lead to errors/bugs down the road).

Reply:
I spoke with Emilia and since the subscriptionStatus is older code, we'll leave as-is for now.

Reply:
Long term I think the subscriptionStatus should be refactored and we can address it then.